core/Inst.hpp

```diff
@@ -163,6 +189,9 @@ namespace olympia
             return dest_reg_bit_masks_[rf];
         }
 
+        RenameData & getRenameData(){
+            return rename_data;
+        }
```

Comment:
Add a `const` version of this method:
```suggestion
        const RenameData & getRenameData() const {
            return rename_data;
        }
```

Reply:
You resolved, but didn't fix...?

Reply:
Sorry, I missed this comment. Addressed in [d23e796](https://github.com/riscv-software-src/riscv-perf-model/pull/40/commits/d23e796d1caee1fe33b8a2597d38461775489107)